src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -303,13 +306,20 @@ public List<JobModel> enumerateJobs(
       filter.addFilterFlightClass(FlightFilterOp.EQUAL, className);
     }
 
+    HashSet<String> jobIdSet = new HashSet<>(ListUtils.emptyIfNull(jobIds));
```

Comment:
Is this better than using `Objects.requireNonNull()`? Also I would use the more general type `Set` for the variable declaration, (or maybe `var`).
```suggestion
    Set<String> jobIdSet = new HashSet<>(Objects.requireNonNullElse(jobIds, List.of()));
```